setup.cfg

```diff
@@ -48,6 +50,7 @@ test =
     pytest >=5.3
     pytest-cov
     %(acq)s
+    %(mat)s
     %(style)s
 interfaces =
```

Comment:
Could you move interfaces before tests and just change acq and mat in tests to "interfaces"?

Reply:
You sure? I put this here because test were failing

Reply:
Yes, if you add %(mat)s to the interfaces and move it before tests, it will work.

```
interfaces =
    %(acq)s
    %(mat)s
test =
    pytest >=5.3
    pytest-cov
    %(interfaces)s
    %(style)s
```

In fact, I will push a little patch to change the way we install phys2bids in CircleCI later, but for the moment, this would be better.